packages/builders/src/interactions/commands/chatInput/options/attachment.ts

```diff
@@ -0,0 +1,11 @@
+import { ApplicationCommandOptionType } from 'discord-api-types/v10';
+import { ApplicationCommandOptionBase } from './ApplicationCommandOptionBase.js';
+
+/**
+ * A chat input command attachment option.
+ */
+export class ChatInputCommandAttachmentOption extends ApplicationCommandOptionBase {
```

Comment:
Shouldn't this be suffixed with `Builder`?

Reply:
They weren't before either, shrug.